hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -238,6 +238,8 @@ public static TypedProperties flinkConf2TypedProperties(Configuration conf) {
         properties.put(option.key(), option.defaultValue());
       }
     }
+    properties.put(HoodieMemoryConfig.MAX_MEMORY_FOR_MERGE_PROP,
```

Comment:
if put property in this way, it means when adding other options in further, we had to add code accordingly here, would we change the config key `write.merge.max_memory` to `hoodie.memory.merge.max.size` to simplify the config transformation here?